README.md

```diff
@@ -107,6 +107,33 @@ While `mamba` and `micromamba` are generally a drop-in replacement for `conda` t
 
 Please refer to the instructions given by the [official documentation](https://mamba.readthedocs.io/en/latest/developer_zone/dev_environment.html).
 
+## API and ABI stability
+
+The Mamba project uses semantic versioning of the form `MAJOR.MINOR.PATCH`.
```

Comment:
We should mention that mamba has two versions:
 - one for the API following semantic versioning
 - one for the ABI following [libtool's convention](https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html#Updating-version-info)
 
And potentially point to their definitions: 

https://github.com/mamba-org/mamba/blob/1ac3174e3864194e9e971b69eb83b3a7f068e565/libmamba/CMakeLists.txt#L16-L56

Reply:
Good point, though I'm not super aware of what we do for ABI versioning. Do you feel like suggesting a paragraph?